ydb/apps/ydb/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 * Fixed a bug in TPC-H tables schema where the `partsupp` table had incorrect list of key columns
-
 * Enhanced parallelism of data restoring in `ydb tools restore`
+* Fixed a bug that `--upload-batch-bytes` when it is set to maximum returns `Too much data` in `ydb tools restore`
```

Comment:
Fixed a bug where `ydb tools restore` could fail with `Too much data` when a batch size was equal to a value in `--upload-batch-bytes` option